src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravelDate/TravelDateSheet.tsx

```diff
@@ -61,8 +67,26 @@ export const TravelDateSheet = forwardRef<ScrollView, Props>(
           ref={focusRef}
           centerContent={true}
         >
+          {maximumDate && (
+            <Section withBottomPadding>
```

Comment:
`Section` should only be used with section item components. Should rather just pass a `style` to the message box if we want some bottom margin.